README.md

```diff
@@ -455,6 +497,16 @@ For detailed information about the custom metrics, see [Custom Metrics Documenta
 kubectl port-forward svc/kube-prometheus-stack-grafana 3000:80 -n inferno-autoscaler-monitoring
 ```
 
+## Documentation
+
+### HPA Integration
+- [HPA Integration Quick Start](docs/hpa-integration.md) - Setup HPA with Inferno optimization signals
+
+### Additional Resources
+- [Custom Metrics Documentation](docs/custom-metrics.md) - Understanding Inferno's metrics system
+- [CRD Documentation](docs/crd-docs.md) - VariantAutoscaling custom resource details
+- [Modeling & Optimization](docs/modeling-optimization.md) - Deep dive into optimization algorithms
+
```

Comment:
I think you should remove this section from readme but please keep the design docs in the docs folder.